src/pages/Form/Form.tsx

```diff
@@ -1,21 +1,25 @@
-import React, { FC, useState } from 'react';
+import React, { FC } from 'react';
+import { useSelector } from 'react-redux';
+import useActions from '../../redux/hooks/useActions';
+import { RootState } from '../../redux/store';
 import OrderForm from '../../components/OrderForm';
 import FormCardsList from '../../components/FormCardsList';
 import { IFormCard } from '../../types/interfaces';
 import styles from './Form.module.scss';
 
 const Form: FC = (): JSX.Element => {
-  const [items, setItems] = useState<IFormCard[]>([]);
+  const formCardList = useSelector<RootState, IFormCard[]>((state) => state.form);
```

Comment:
u can create additinal types file to add:
```
import { useDispatch, useSelector } from 'react-redux'
import type { TypedUseSelectorHook } from 'react-redux'
import type { RootState, AppDispatch } from './store'

// Use throughout your app instead of plain `useDispatch` and `useSelector`
export const useAppDispatch: () => AppDispatch = useDispatch
export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector
```
and use selectors with types